### PR TITLE
Remove duplication in TypeConverter.Tests.builds

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

**Comment by alexperovich** [comment]
@weshaggard

File: src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.builds [hunk 1]
```diff
@@ -16,10 +16,6 @@
       <TestTFMs>net462</TestTFMs>
```
Comment (weshaggard) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.builds]: Do we need the net462 build? I suspect we don't. @Priya91?

Comment (Priya91) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.builds]: Yeah we dont, net463 is built.

File: src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj [hunk 2]
```diff
@@ -87,7 +87,7 @@
     </ProjectReference>
   </ItemGroup>
   <!-- TODO: Replace this with a package reference. -->
-  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.7'">
+  <ItemGroup>
     <ProjectReference Include="..\..\System.Drawing.Primitives\pkg\System.Drawing.Primitives.pkgproj" />
   </ItemGroup>
```
Comment (Priya91) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: why Drawing.Primitives..

Comment (alexperovich) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: For Color/Point and others

Comment (Priya91) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj]: ok cool.

**Comment by Priya91** [comment]
LGTM

File: src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.builds (hunk 1 above)
Comment (weshaggard) [inline src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.builds]: OK we should probably update to remove the net462 build configuration.

**Pull request merged**